### PR TITLE
Phpamqp driver implementation

### PR DESCRIPTION
**Pull request by acrobat**
Move the phpamqp driver to the new namespace

See #256

### CODE REVIEW

**Comment by acrobat** [comment]
Is anything more here needed? I guess custom composer file and travis config etc will be done in the separate repository?

File: src/Driver/PhpAmqp/Driver.php [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <?php
 
-namespace Bernard\Driver;
+namespace Bernard\Driver\PhpAmqp;
```
Comment (sagikazarmark) [inline src/Driver/PhpAmqp/Driver.php]: Wonder if simply `Amqp` would be a better name.

File: tests/Driver/PhpAmqp/DriverTest.php [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 use PhpAmqpLib\Connection\AMQPStreamConnection;
 use PhpAmqpLib\Message\AMQPMessage;
+use PHPUnit\Framework\TestCase;
 
-class PhpAmqpDriverTest extends \PHPUnit\Framework\TestCase
+final class DriverTest extends TestCase
 {
     /**
-     * @var AMQPStreamConnection
+     * @var AMQPStreamConnection|\PHPUnit_Framework_MockObject_MockObject
```
Comment (sagikazarmark) [inline tests/Driver/PhpAmqp/DriverTest.php]: Isn't this still the legacy object name? As of 6.0 there should be properly namespaced class names.

**Comment by sagikazarmark** [comment]
I would probably create a branch and merge every driver change there before merging to master.

**Comment by acrobat** [comment]
ah yes, good point! I will change the base branch to the new drivers branch, so we can group all changes there!

File: tests/Driver/PhpAmqp/DriverTest.php (hunk 2 above)
Comment (acrobat) [reply tests/Driver/PhpAmqp/DriverTest.php]: Namespace support for the mockobject lib was just merged sebastianbergmann/phpunit-mock-objects#376. So in a next version we will be able to replace this!

File: src/Driver/PhpAmqp/Driver.php (hunk 1 above)
Comment (acrobat) [reply src/Driver/PhpAmqp/Driver.php]: Now would be indeed a good time to rename this. Will do!

**Comment by acrobat** [comment]
@sagikazarmark you can merge this when you want! I will create some more PR's against this branch

**Pull request merged**